{
    "type": "bundle",
    "id": "bundle--006ae059-32b9-40b6-851e-69b21529ae84",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--405b0e30-efa0-4ed0-8b98-5399943b597d",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-05-13T00:42:55.719373Z",
            "modified": "2025-05-13T00:42:55.719373Z",
            "name": "CVE-2025-31195",
            "description": "The issue was addressed by adding additional logic. This issue is fixed in macOS Sequoia 15.4. An app may be able to break out of its sandbox.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-31195"
                }
            ]
        }
    ]
}